### PR TITLE
Adds safety in batch calls logic

### PR DESCRIPTION
**Pull request by Berardpi**
Also renames application sample template file

### CODE REVIEW

**Comment by Berardpi** [comment]
@MarcoCode Can you take a look?

File: app/models/maestrano/connector/rails/concerns/complex_entity.rb [hunk 1]
```diff
@@ -27,8 +27,8 @@ def formatted_entities_names(names)
       names.index_by { |name| name }.with_indifferent_access
     end
 
-    def count_entities(entities)
-      entities.values.map(&:size).max
+    def count_and_first(entities)
```
Comment (MarcoCode) [inline app/models/maestrano/connector/rails/concerns/complex_entity.rb]: Might be worth adding a comment to point out that for a complex_entity is get biggest array and count?

**Comment by MarcoCode** [comment]
👍

**Comment by Berardpi** [comment]
Done, thanks for the review @MarcoCode

**Pull request merged**